custom_components/waste_collection_schedule/waste_collection_schedule/source/ittre_be.py

```diff
@@ -0,0 +1,108 @@
+import re
+from datetime import date, datetime
+
+import requests
+from bs4 import BeautifulSoup
+from dateutil.rrule import FR, MO, SA, SU, TH, TU, WE, WEEKLY, rrule
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Ittre"
+DESCRIPTION = "Source for Ittre."
+URL = "https://www.ittre.be/"
+TEST_CASES: dict[str, dict] = {"Ittre": {}}
+
+
+ICON_MAP = {
+    "Trash": "mdi:trash-can",
+    "Glass": "mdi:bottle-soda",
+    "Bio": "mdi:leaf",
+    "Paper": "mdi:package-variant",
+    "Recycle": "mdi:recycle",
+}
+
+
+FRENSH_DAYS = {
+    "lundi": MO,
```

Comment:
I would add:
`"lundis": MO,`
because the page says "tous les lundis" with an 's'